SharedPackages/BrowserServicesKit/Sources/Subscription/Managers/V1toV2Bridging/SubscriptionAuthV1toV2Bridge.swift

```diff
@@ -25,14 +25,21 @@ import os.log
 /// Temporary bridge between auth v1 and v2, this is implemented by SubscriptionManager V1 and V2
 public protocol SubscriptionAuthV1toV2Bridge: SubscriptionTokenProvider, SubscriptionAuthenticationStateProvider {
 
-    /// If the feature is enabled in the app, based on Subscription entitlements
-    /// This is mostly used by the UI for showing features and their state
-    func isFeatureAvailableAndEnabled(feature: Entitlement.ProductName, cachePolicy: APICachePolicy) async throws -> Bool
-    /// If the user is allowed to use the feature, base on the TokenContainer entitlements
-    /// This is used by VPN and PIR
-    func isFeatureEnabledForUser(feature: Entitlement.ProductName) async throws -> Bool
-
-    func currentSubscriptionFeatures() async -> [Entitlement.ProductName]
+    /// Whether a feature is included in the Subscription.
+    ///
+    /// This is mostly useful pre-purchase so that the user will know which features the Subscription includes for them.
+    /// This should really not be used for much else, as the client UI should check ``isFeatureAvailable``
+    /// to know if a feature can be used by the user.
+    ///
+    func isFeatureIncludedInSubscription(_ feature: Entitlement.ProductName, cachePolicy: APICachePolicy) async throws -> Bool
```

Comment:
I think this method should possibly not throw and return an expired cache if necessary.

Reply:
@diegoreymendez Should I implement the cache while you are looking at the rest??

Reply:
@federicocappelli - Not sure tbh... I think we need to first decide what we want from our cache.  Let's chat.